.env.example

```diff
@@ -3,17 +3,17 @@ APP_COMPONENT=api
 APP_HOST=0.0.0.0
 APP_PORT=80
 LOG_LEVEL=10
-READ_DB_DRIVER=mysql
-READ_DB_USER=root
+READ_DB_DRIVER=postgres
```

Comment:
```suggestion
READ_DB_DRIVER=postgresql
```

Reply:
thanks for this one @retrocraft lol

Reply:
LMAO i can't believe nobody noticed until they made a breaking change over a decade later